pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -62,3 +62,15 @@ rustPlatform.buildRustPackage rec {
     maintainers = with maintainers; [ pamplemousse pimeys superherointj ];
   };
 }
+
+### Troubleshooting
+# Here's an example application using Prisma with Nix: https://github.com/pimeys/nix-prisma-example
+# At example's `flake.nix` shellHook, notice the requirement of defining environment variables for prisma, it's values will show on `prisma --version`.
+# Read the example's README: https://github.com/pimeys/nix-prisma-example/blob/main/README.md
+# Prisma requires 2 packages, `prisma-engines` and `nodePackages.prisma`, to be at *exact* same versions.
+# Certify at `package.json` that dependencies "@prisma/client" and "prisma" are equal, meaning no caret (`^`) in version.
+# Configure NPM to use exact version: `npm config set save-exact=true`
+# Delete `package-lock.json` and rebuild it by running `npm install`.
```

Comment:
```suggestion
# Delete `package-lock.json`, delete `node_modules` folder and run `npm install`.
```

Reply:
fixed, but instead of using the word `folder` we can use `directory`...